README.md

```diff
@@ -1,4 +1,4 @@
-# TwigView plugin for CakePHP #
+# TwigView plugin for CakePHP 4.0+
```

Comment:
I would keep the version out of it for readmes wherever possible. 
this often gets outdated too quickly.